### PR TITLE
travis-ci: enable ccache for build speedup

### PR DESCRIPTION
**Pull request by chipitsine**
Changes proposed in this pull request:
 - 
 - 
 - 

Your great patch is much appreciated. We are considering to apply your patch into the SoftEther VPN main tree.

SoftEther VPN Patch Acceptance Policy:
http://www.softether.org/5-download/src/9.patch

You have two options which are described on the above policy.
Could you please choose either option 1 or 2, and specify it clearly on the reply?

-

option 1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,14 @@ language: c
 
 compiler: [ gcc, clang ]
 
+cache:
+  ccache: true
```
Comment (moatazelmasry2) [inline .travis.yml]: I don't know much about travis-ci but according to the documentation, it is defined as "cache: ccache"
https://docs.travis-ci.com/user/caching/

Comment (moatazelmasry2) [reply .travis.yml]: Also there's the comment:
"to cache $HOME/.ccache and automatically add /usr/lib/ccache to your $PATH."

Comment (chipitsine) [reply .travis.yml]: That way of specifying cache is limited to ccache only

If we ever will want to add other caches, "cache: ccache" will not work

As for now we can use short form, you are right

**Comment by chipitsine** [comment]
closing in favour of https://github.com/SoftEtherVPN/SoftEtherVPN/pull/462

**Pull request closed without merging**